classes/local/rule_checker.php

```diff
@@ -186,8 +274,14 @@ public function get_validation_messages(): array {
         }
         return $messages;
     }
-    
-    public function add_error_field($mform) {
+
+    /**
+     * Inject a form element to append information about failed checks as validation errors to.
```

Comment:
Maybe... "Inject a form element so we can append information about failed checks to it as validation errors."